dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -4202,10 +4272,11 @@ void Spdp::SpdpTransport::thread_status_task(const DCPS::MonotonicTimePoint& /*n
     StatusMap running;
     StatusMap removed;
     if (!local_thread_status_manager_.sync_with_parent(
-          *global_thread_status_manager_, running, removed) &&
-        DCPS::DCPS_debug_level) {
-      ACE_ERROR((LM_ERROR, "(%P|%t) ERROR: Spdp::SpdpTransport::thread_status_task: "
-        "syncing with global thread status manager failed\n"));
+          *global_thread_status_manager_, running, removed)) {
+      if (DCPS::DCPS_debug_level > 0) {
```

Comment:
Why is this changing?  The change doesn't seem to have any effect.

Reply:
The return was done in the past only when debug level was set

Reply:
Right, I didn't see that detail before.

Reply:
This was tricky code, took me also some time to see the initial issue when checking the debug log level in this file